test/runtime/Policies.go

```diff
@@ -1139,6 +1139,122 @@ var _ = Describe("RuntimePolicies", func() {
+		}
+
+		// curlWithRetry retries the curl, to make sure that allowed curls don't
+		// flake on bad connectivity
+		curlWithRetry := func(name string, cmd string, optionalArgs ...interface{}) (res *helpers.CmdRes) {
+			for try := 0; try < 5; try++ {
+				res = vm.ContainerExec(name, helpers.CurlFail(cmd, optionalArgs...))
+				if res.WasSuccessful() {
+					return res
+				}
+			}
+			return res
+		}
+
+		By("setting policy enforcement to default so that EntityNone is the source of deny-all")
+		ExpectPolicyEnforcementUpdated(vm, helpers.PolicyEnforcementDefault)
+		app1Label := fmt.Sprintf("id.%s", helpers.App1)
+		policy := fmt.Sprintf(`
+		[{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toEntities": [
+					"%s"
+				]
+			}]
+		}]`, app1Label, api.EntityNone)
+		setupPolicy(policy)
+
+		app2Label := fmt.Sprintf("id.%s", helpers.App2)
+		policy = fmt.Sprintf(`
+		[{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toEntities": [
+					"%s"
+				]
+			}]
+		}]`, app2Label, api.EntityNone)
+		setupPolicy(policy)
+
+		By("testing that EntityNone is denying all egress for app1 and app2")
+		vm.ContainerExec(helpers.App1, helpers.CurlFail("http://%s/public", http1IP[helpers.IPv4])).ExpectFail("%q can make http request to pod", helpers.App1)
+		vm.ContainerExec(helpers.App1, helpers.CurlFail("-4 http://%s", worldIP)).ExpectFail("%q can make http request to %s", helpers.App1, worldIP)
+		vm.ContainerExec(helpers.App2, helpers.CurlFail("http://%s/public", http1IP[helpers.IPv4])).ExpectFail("%q can make http request to pod", helpers.App1)
+		vm.ContainerExec(helpers.App2, helpers.CurlFail("-4 http://%s", worldIP)).ExpectFail("%q can make http request to %s", helpers.App1, worldIP)
+
+		By("testing basic egress between endpoints (app1->app2)")
+		policy = fmt.Sprintf(`
+		[{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toEndpoints": [{"matchLabels": {"%s": ""}}]
+			}]
+		}]`, app1Label, httpd1Label)
+		setupPolicy(policy)
+		curlWithRetry(helpers.App1, "http://%s/public", http1IP[helpers.IPv4]).ExpectSuccess("%q cannot make http request to pod", helpers.App1)
+		vm.ContainerExec(helpers.App1, helpers.CurlFail("-4 http://%s", worldIP)).ExpectFail("%q can make http request to %s", helpers.App1, worldIP)
+
+		By("testing basic egress to 1.1.1.1/32")
+		policy = fmt.Sprintf(`
+		[{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toCIDR": [
+					"1.1.1.1/32"
+				]
+			}]
+		}]`, app1Label)
+		setupPolicy(policy)
+		curlWithRetry(helpers.App1, "http://%s/public", http1IP[helpers.IPv4]).ExpectSuccess("%q cannot make http request to pod", helpers.App1)
+		curlWithRetry(helpers.App1, "-4 http://%s", worldIP).ExpectSuccess("%q cannot make http request to pod", helpers.App1)
+
+		By("testing egress toEntity: world in combination with previous rules (app1)")
```

Comment:
You use `api.EntityAll` instead of `api.EntityWorld` in the below policy.

Reply:
Oops, good catch. One of these was supposed to be EntityWorld and the other EntityAll. I'll fix this up and make the `By` more explicit.